modin/pandas/dataframe.py

```diff
@@ -1421,7 +1422,7 @@ def memory_usage(self, index=True, deep=False):  # noqa: PR01, RT01, D200
                 self._query_compiler.memory_usage(index=False, deep=deep)
             )
             index_value = self.index.memory_usage(deep=deep)
-            return Series(index_value, index=["Index"]).append(result)
+            return modin.pandas.concat([Series(index_value, index=["Index"]), result])
```

Comment:
this feels weird, cannot we instead do `from modin.pandas.general import concat` instead?..

Reply:
@vnlitvinov `from modin.pandas.general import concat` gives

`ImportError: cannot import name 'DataFrame' from partially initialized module 'modin.pandas.dataframe' (most likely due to a circular import) (/Users/maheshvashishtha/software_sources/modin/modin/pandas/dataframe.py)`

and `from modin.pandas import concat` gives

`ImportError: cannot import name 'concat' from partially initialized module 'modin.pandas' (most likely due to a circular import) (/Users/maheshvashishtha/software_sources/modin/modin/pandas/__init__.py)`

I now have `import modin.pandas as pd` and `pd.concat` instead.

Reply:
Never mind. `import modin.pandas as pd` gives `AttributeError: module 'modin' has no attribute 'pandas'` on python 3.6: https://github.com/modin-project/modin/actions/runs/3202049341/jobs/5230663043

I reverted to `import modin` and calling `modin.pandas.concat`.